README.md

```diff
@@ -23,11 +23,16 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   
 ### Application can work at two modes:
 -  Local mode
-  -  All the ROS nodes run at the same machine.
+  -  All the functions run at the same machine.
 -  Network mode
-  -  ROS nodes run at two different machine: Server and Client.
-  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
-  -  Server runs all other ROS nodes except two nodes running on client.
+  -  Functions are distributed to two different machine: Server and Client.
+  -  Server Machine:
+    -  Runs perceptions nodes such as surface detection
+    -  Runs motion planning nodes such as the process path planner and free motion planner (moveit).
+  -  Client Machine:
+    -  Runs Rviz for user front-end and visualization only.
+    -  Runs either industrial_robot_simulator node or robot_driver node depending on whether it is on sim or real mode.
+    -  Runs either generate_point_cloud node or Kinect sensor driver node also based on the mode being used.
 
```

Comment:
:smile:  Yours is much better than I did before, so I decided to use yours directly.

How about we change like below: 

-  Network mode
  -  Functions are distributed to two different machines: Server and Client.
  -  Server Machine:
    -  Runs perceptions nodes such as surface detection.
    -  Runs motion planning nodes such as the process path planner and free motion planner (moveit).
  -  Client Machine:
    -  Runs Rviz for user front-end and visualization only.
    -  Runs industrial_robot_simulator node when simulated robot is being used.
    -  Runs robot_driver node when real robot is being used.
    -  Runs generate_point_cloud node when simulated sensor is being used.
    -  Runs Kinect sensor driver node when real sensor is being used.